a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
```

Comment:
Ahh, I see now. Since mixin's suck, can we just put those attributes into A10Base ?